cli/runtime/Makefile

```diff
@@ -17,6 +17,20 @@ GOBIN=$(shell go env GOBIN)
 endif
 GO := go
 
+NUL = /dev/null
+ifeq ($(GOHOSTOS),windows)
+	NUL = NUL
+endif
+
+BUILD_SHA ?= $$(git describe --match=$(git rev-parse --short HEAD) --always --dirty)
```

Comment:
nit: are BUILD_SHA/DATE used?

Reply:
Both of them are used to assign it to the build time variable configuration. Not sure actual variables are used in the code or not. But if not, I think we should show this things as part of `tanzu version` information.